application/routers/entity.py

```diff
@@ -338,6 +353,15 @@ def search_entities(
     )
 
 
+def convert_to_model(data):
```

Comment:
The Entity factory already. roughly does this. I think. this can. be removed if you remove. the code from the entity search function